src/components/newCardForm.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+
+function NewCardForm({ setCards, setShowForm, boards, selectedBoard }) {
+const [text, setText] = useState("");
+const [board, setBoard] = useState("");
+
+const handleSubmit = (event) => {
+    event.preventDefault();
+
+    if (text && board) {
+    const newCard = {
+        id: Math.round(Math.random() * 10000000),
+        text,
+        board,
+        votesInteresting: 0,
+    };
+    setCards((cards) => [newCard, ...cards]);
```

Comment:
👀 This isn't adding a new card to the database. This will only affect the local state, and any cards added will be lost when the page reloads.

Rather than passing in `setCards` from `App`, we should pass in a function that the form can use to notify `App` that there is new card data available. `App` would use that data to both create the new card on the backend, as well as adding the newly created card to the frontend state. The _backend_ would be responsible for assigning an id to the new card (or rather, the database will when the backend adds the card data to the database).